nautobot/ui/src/views/generic/ObjectRetrieve.js

```diff
@@ -305,74 +320,119 @@ export default function ObjectRetrieve({ api_url }) {
-                                                        alignItems="center"
-                                                    >
-                                                        <NtcThumbnailIcon
-                                                            width="25px"
-                                                            height="30px"
-                                                        />
-                                                        &nbsp;
-                                                        {render_header(
+                                        <NautobotGridItem colSpan={4}>
+                                            <Heading
+                                                display="flex"
+                                                alignItems="center"
+                                            >
+                                                <NtcThumbnailIcon
+                                                    width="25px"
+                                                    height="30px"
+                                                />
+                                                &nbsp;
+                                                {render_header(
+                                                    objectData.display
+                                                )}
+                                            </Heading>
+                                            <br />
+                                            <TableContainer>
+                                                <Table>
+                                                    <Tbody>
+                                                        {Object.entries(
+                                                            objectData
+                                                        ).map((key, idx) => (
+                                                            <RenderRow
+                                                                identifier={
+                                                                    key[0]
+                                                                }
+                                                                value={key[1]}
+                                                                advanced={false}
+                                                                key={`${key}_${idx}`}
+                                                            />
+                                                        ))}
+                                                    </Tbody>
+                                                </Table>
+                                            </TableContainer>
+                                        </NautobotGridItem>
+                                    </NautobotGrid>
+                                </Card>
+                            </TabPanel>
+                        ) : (
```

Comment:
Instead of these massive ternary statements, can we extract some of this logic/structure into a few helper functions?